### PR TITLE
:recycle: streamlining the codebase

### LINKED ISSUE #279: update navbar with correct links

### PR DESCRIPTION
**Pull request by eliasnijs**
closes #279

### CODE REVIEW

File: frontend/src/components/navbar/Navbar.jsx [hunk 1]
```diff
@@ -35,8 +34,8 @@ export default function Navbar() {
         <LinkList
           name={"Menu"}
           categories={{
-            Planning: { icon: faCalendarWeek, link: "/beheer/planning" },
-            "Nieuwe data": {
+            Dashboard: { icon: faCalendarWeek, link: "/beheer/dashboard" },
+            "Data beheer": {
```
Comment (JasperJanin) [inline frontend/src/components/navbar/Navbar.jsx:38]:
```suggestion
            "Databeheer": {
```

File: frontend/src/components/navbar/Navbar.jsx [hunk 2]
```diff
@@ -47,7 +46,7 @@ export default function Navbar() {
         <LinkList
           name={"Data"}
           categories={{
-            Rondes: { icon: faBicycle, link: "#" },
+            Planningen: { icon: faBicycle, link: "/beheer/planningen/1" },
```
Comment (JasperJanin) [inline frontend/src/components/navbar/Navbar.jsx:49]: This breaks when this schedule is ever removed

**Review by JasperJanin — CHANGES_REQUESTED** [review]
This appears to contain a breaking change

File: frontend/src/components/navbar/Navbar.jsx (hunk 2 above)
Comment (eliasnijs) [reply frontend/src/components/navbar/Navbar.jsx:49]: This is intended. If we want a decent fix for this we need to do it in a larger PR. At the moment there are multiple cases such as this one.

**Review by JasperJanin — APPROVED** [review]

**Pull request merged**